docs/content/3.forms/10.form.md

```diff
@@ -0,0 +1,325 @@
+---
+description: Collect and validate form data.
+links:
+  - label: GitHub
+    icon: i-simple-icons-github
+    to: https://github.com/nuxtlabs/ui/blob/dev/src/runtime/components/forms/Form.ts
+navigation:
+  badge: Edge
+---
+
+## Usage
+
+Use the Form component to validate form data using schema libraries such as [Yup](https://github.com/jquense/yup), [Zod](https://github.com/colinhacks/zod), [Joi](https://github.com/hapijs/joi) or your own validation logic. It works seamlessly with the FormGroup component to automatically display error messages around form elements.
+
+The Form component requires the `validate` and `state` props for form validation.
+
+- `state` - a reactive object that holds the current state of the form.
+- `validate` - a function that takes the form's state as input and returns an array of `FormError` objects with the following fields:
+  - `message` - the error message to display.
+  - `path` - the path to the form element matching the `name`.
```

Comment:
Is there a reason not to use `name` directly instead of `path` here?

Reply:
I thought about this too when writing the documentation, I'm a bit divided here, the term `path` seems to be the common name in schema / form validation libraries (it's used in zod, yup, joi, vuelidate, naiveui...), but mixing it with `name` is a bit confusing. 

I'd stick to it to stay consistent, and rollback the `name` prop to `path` on UFormGroup for consistency.

Reply:
Ok let's keep it this way, but are you sure it's necessary to add a `path` prop the the `FormGroup`? It adds an extra prop that need to be written each time and will in most case always be equal to `name`.

Reply:
I agree, let's keep it like this, I'm overthinking it. We can still introduce the `path` prop later if users find it confusing.